kotlinx-collections-immutable/tests/src/test/kotlin/kotlinx.collections.immutable/contractTests/immutableSet/ImmutableSetTest.kt

```diff
@@ -24,15 +24,22 @@ import kotlin.test.*
 
 class ImmutableHashSetTest : ImmutableSetTestBase() {
     override fun <T> immutableSetOf(vararg elements: T) = persistentHashSetOf(*elements)
+    override fun <T> testBuilderToPersistentSet(builder: PersistentSet.Builder<T>) {
```

Comment:
Makes sense to add a similar test for `toPersistentMap`

Reply:
+1